<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1027</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Shredder</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>999</width>
      <height>679</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {background-color : white; }</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1027</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuFilter">
    <property name="title">
     <string>Filter</string>
    </property>
    <widget class="QMenu" name="menu_Channels">
     <property name="title">
      <string>&amp;Channels</string>
     </property>
     <addaction name="actionRed"/>
     <addaction name="actionGreen"/>
     <addaction name="actionBlue"/>
    </widget>
    <addaction name="actionInvert"/>
    <addaction name="actionGrayscale"/>
    <addaction name="actionEdge_Detect"/>
    <addaction name="actionFace_Detect"/>
    <addaction name="actionEye_Detect"/>
    <addaction name="actionRed_Eye_Removal"/>
    <addaction name="menu_Channels"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuFilter"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionInvert">
   <property name="text">
    <string>Invert</string>
   </property>
  </action>
  <action name="actionGrayscale">
   <property name="text">
    <string>Grayscale</string>
   </property>
  </action>
  <action name="actionRed_ish">
   <property name="text">
    <string>&amp;Edge Detect</string>
   </property>
  </action>
  <action name="actionBlue_ish">
   <property name="text">
    <string>Blue-ish</string>
   </property>
  </action>
  <action name="actionGreen_ish">
   <property name="text">
    <string>Green-ish</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionAbout_OpenCV">
   <property name="text">
    <string>About OpenCV</string>
   </property>
  </action>
  <action name="actionEdge_Detect">
   <property name="text">
    <string>&amp;Edge Detect</string>
   </property>
  </action>
  <action name="actionRed">
   <property name="text">
    <string>&amp;Red</string>
   </property>
  </action>
  <action name="actionGreen">
   <property name="text">
    <string>&amp;Green</string>
   </property>
  </action>
  <action name="actionBlue">
   <property name="text">
    <string>&amp;Blue</string>
   </property>
  </action>
  <action name="actionFace_Detect">
   <property name="text">
    <string>Face Detection</string>
   </property>
  </action>
  <action name="actionEye_Detect">
   <property name="text">
    <string>&amp;Eye Detection</string>
   </property>
  </action>
  <action name="actionRed_Eye_Removal">
   <property name="text">
    <string>&amp;Red Eye Removal</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
